sys/Makefile

```diff
@@ -12,13 +12,22 @@ CLEAN-FILES += mimiker.elf.map
 
 include $(TOPDIR)/build/build.kern.mk
 
-LDFLAGS += -T $(TOPDIR)/sys/$(ARCH)/$(ARCH).ld
+LDFLAGS += -T $(TOPDIR)/sys/kernel.ld
```

Comment:
I don't like this change. It's architecture specific so from my point of view it should lives in architecture directory.

Reply:
I agree. It's been moved to architecture directory.